docker-compose.yml

```diff
@@ -11,6 +11,18 @@ services:
             - "5433:5432"
         networks:
             - products_network
+
+    mongodb:
+        restart: always
+        image: mongo
```

Comment:
we should specify image version here, latest is not a good choice for a version :D